provisio-maven-plugin/src/main/java/ca/vanzyl/maven/plugins/provisio/ProvisioningMojo.java

```diff
@@ -35,10 +35,17 @@
 @Mojo(name = "provision", defaultPhase = LifecyclePhase.PACKAGE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class ProvisioningMojo extends BaseMojo {
 
+  @Parameter(defaultValue = "false", property = "skipProvision")
```

Comment:
Should it not just be "skip" and have the namespace from the goal .. similar to maven.test.skip or whatever

Reply:
I was following skipTests from surefire plugin.

Reply:
Cool